app/providers/list/page.tsx

```
@@ -12,39 +11,26 @@ import {
   PaginationNext,
   PaginationEllipsis
 } from "@/components/ui/pagination";
+import { 
+  PROVIDER_API, 
+  Provider, 
+  ProvidersListResponse, 
+  ApiProvider, 
```

ESLint

[error] 'ApiProvider' is defined but never used.

(@typescript-eslint/no-unused-vars)